app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -56,10 +68,46 @@ export default function NoteTable({
               </td>
               <td>
                 <div className="flex flex-row items-center justify-start text-gray-900">
-                  {note.title}
+                  <a>{note.title}</a>
+                </div>
+              </td>
+              <td>
+                {note.description.length > 20
+                  ? note.description.slice(0, 20) + "..."
+                  : note.description}
+              </td>
+              <td className="text-center">
+                {note.tags.map(tag => (
+                  <label key={tag.id}>
+                    <Badge color={tag.color}>{tag.label}</Badge>
+                  </label>
+                ))}
+              </td>
+              <td>
+                <label>{note.created_date}</label>
+              </td>
+              <td>
+                <label>{note.due_date ? note.due_date : "--"}</label>
+              </td>
+              <td>
+                <label>
+                  <Avatar size={36} contact={{ name: note.contact }} />
+                </label>
+              </td>
+              <td>
+                <div className="flex flex-row space-x-3">
+                  <Tooltip
+                    content={<span>Edit</span>}
+                    position="bottom"
+                    className="ml-auto"
+                  >
+                    <i className="ri-pencil-line ri-lg text-gray-600 hover:text-gray-900"></i>
+                  </Tooltip>
+                  <Tooltip content={<span>Delete</span>} position="bottom">
+                    <i className="ri-delete-bin-5-line ri-lg text-gray-600 hover:text-gray-900"></i>
+                  </Tooltip>
```

Comment:
Instead of using `i` tag, please use [neetoUI Icon Button](https://neeto-ui.netlify.app/#/button).